woocommerce/woocommerce
# Add product management description

### All Submissions:






### Changes proposed in this Pull Request:

Adds a description field to the new product management experience.

Also adds `context` to the API request to fix up returned response data.

**New editor:**
<img width="546" alt="Screen Shot 2022-10-05 at 10 11 07 AM" src="https://user-images.githubusercontent.com/10561050/194121964-e4b6f7cd-b1ba-4524-b805-76894c8a8f73.png">

**Old editor:**
<img width="408" alt="Screen Shot 2022-10-05 at 10 11 22 AM" src="https://user-images.githubusercontent.com/10561050/194121959-f6a4642a-ebae-4444-bdb5-6f44b53c8dbf.png">

### How to test the changes in this Pull Request:

1. Go to the new product management experience `Products -> Add new (MVP)`
2. And add some content to the description field (note there are still some odd quirks around block selection in the editor that are unrelated to this PR)
3. Save the content
4. Refresh and make sure it works as expected
5. Visit the old product edit page for the same product
6. Note that content works as expected

### Other information:




### FOR PR REVIEWER ONLY:
